argilla/docs/reference/argilla/settings/task_distribution.md

```diff
@@ -0,0 +1,21 @@
+---
+hide: footer
+---
+# Task Distribution
+
+Task Distribution settings are used to define the criteria used by the tool to automatically manage records in the dataset depending on the expected number of submitted responses per record.
+
+## Usage Examples
+
+The default minimum submitted responses per record is 1. If you wish to increase this value, you can define it through the `TaskDistribution` class and pass it to the `Settings` class.
+
+[CODE SNIPPET]
+
+---
+
+## `rg.VectorField`
+
+::: src.argilla.settings._vector.VectorField
```

Comment:
why vector field?

Reply:
Hi David! Sorry, these are just placeholders. The docs aren't yet ready for review, I just set it as such so the link would be generated for Amélie to check the overall structure